topology/src/main/java/io/camunda/zeebe/topology/changes/TopologyChangeCoordinatorImpl.java

```diff
@@ -213,11 +213,11 @@ private void simulateTopologyChange(
   }
 
   private void failFuture(final ActorFuture<?> future, final Throwable error) {
+    LOG.error("Failed to handle topology request", error);
```

Comment:
:wrench: A warning might be enough. Sometimes the errors are due to user error, like invalid request or new request when there is an ongoing change.